src/onnxruntime.cc

```diff
@@ -239,9 +239,67 @@ ModelState::LoadModel(
               RETURN_IF_ERROR(ea.MemberAsString("name", &name));
 #ifdef TRITON_ENABLE_ONNXRUNTIME_TENSORRT
               if (name == kTensorRTExecutionAccelerator) {
+                // create tensorrt options with default values
+                OrtTensorRTProviderOptions trt_options{
+                    instance_group_device_id,
+                    stream != nullptr ? 1 : 0,
+                    stream != nullptr ? (void*)stream : nullptr,
+                    1000,     // trt_max_partition_iterations
+                    1,        // trt_min_subgraph_size
+                    1 << 30,  // max_workspace_size
+                    0,        // trt_fp16_enable
+                    0,        // trt_int8_enable
+                    nullptr,  // trt_int8_calibration_table_name
+                    0,        // trt_int8_calibration_table_name
+                    0,        // trt_dla_enable
+                    0,        // trt_dla_core
+                    0,        // trt_dump_subgraphs
+                    0,        // trt_engine_cache_enable
+                    nullptr,  // trt_engine_cache_path
+                    0,        // trt_engine_decryption_enable
+                    nullptr,  // trt_engine_decryption_lib_path
+                    0         // trt_force_sequential_engine_build
+                };
+                // Validate and set parameters
+                triton::common::TritonJson::Value params;
```

Comment:
Add a "parameters" section to the README to describe these parameters. For example https://github.com/triton-inference-server/openvino_backend#using-the-openvino-backend

Reply:
thanks! I will add it... I was also looking for a place to document this. Currently I am adding it here: https://github.com/triton-inference-server/server/blob/main/docs/optimization.md#onnx-with-tensorrt-optimization